### PR TITLE
Request for reviewers to review mi code

### PR DESCRIPTION
**Pull request by Stevers1**
Long live microverse.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1 +1,74 @@
...
+    
+  <title>Portfolio</title>
+</head>
+<body>
+  <main>
+    <header class="toolbar">
+      <a href=""><h3 id="home">Home</h3></a>
+
+      <img src="Image_objects/IconoTripleMenu.png" alt="">
+
```
Comment (tresorsawasawa) [inline index.html]: - [ **OPTIONAL** ] Kindly make sure your menu image button is wrapped in a `<nav>` tag as required for the project and for good HTML semantics. Make sure you add this change as it was missed by the previous reviewer

File: index.html [hunk 2]
```diff
@@ -1 +1,74 @@
...
+  <link rel="stylesheet" href="styles.css">
 
+    
+  <title>Portfolio</title>
+</head>
+<body>
+  <main>
+    <header class="toolbar">
+      <a href=""><h3 id="home">Home</h3></a>
+
```
Comment (tresorsawasawa) [inline index.html]: - [ ] Good job making the change required by the previous reviewer. So, I noticed that you didn't update your styling once you added an `<a>` tag(check: the used correct color, no underline decoration on the text,...). Make sure you do so to stick to the Figma design.

File: index.html [hunk 3]
```diff
@@ -1 +1,74 @@
...
+      </div>
+
+      <div id="framew">
+        <h3 class="tskills">Frameworks</h3>
+        <p>Bootstrap</p>
+        <p>Ruby on rails</p>
+        <p>RSPec</p>
+        <p>CapyBara</p>
+
+      </div>
```
Comment (tresorsawasawa) [inline index.html]: - [ ] As required by the previous reviewer kindly make sure each list of **Languages**, **Frameworks**, and **skills** uses a `<ul>` tag to group them, and `<li>` tag per item or element.
- [ ] Also make sure you add the list of **Skils** to all the content of the Figma template.

File: index.html [hunk 4]
```diff
@@ -1 +1,74 @@
...
+
+    <div class="socialmedia">
+      <h3>Social Media</h3>
+      <img src="Image_objects/githubicon.png" alt="">
+      <img src="Image_objects/linkedin.png" alt="">
+      <img src="Image_objects/manita.png" alt="">
+
+    </div>
+  
+
```
Comment (tresorsawasawa) [inline index.html]: - [ ] As mentioned by the previous reviewer, kindly make sure your project sticks to the Fitgma design as required for the project. Correctly check:
    - Fonts(font-size, font-weight, font-family, line-height,...)
    - Colors used on some text
    - Spacing between(margins && paddings) and around content
    - Adding all content(list of skills, some horizontal lines, all icons,...)
    - Add all background decorations and correctly align them
    - etc
### Figma Design 
Homepage 
### Your Design 
Screenshot from 2022-07-01 00-52-59 |

File: styles.css [hunk 5]
```diff
@@ -1 +1,88 @@
...
+  background-position: bottom -100px left;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+}
+
+main {
+  /* border: 5px solid green; */
```
Comment (tresorsawasawa) [inline styles.css]: - [x] Kindly make sure you remove this commented code lines to make your project codebase cleaner

File: README.md [hunk 6]
```diff
@@ -1,2 +1,15 @@
...
+## Built with
+- HTML
+- CSS
+
+## Getting Started
+> Just download or clone my repo and check with an IDE to review my code.
+
+## Author
+
+- GitHub: [@stevers1](https://github.com/Stevers1)
```
Comment (tresorsawasawa) [inline README.md]: - [x] As mentioned by the previous reviewer kindly make sure you use [the Microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md)and make your README more professional. I suggest you take a look at [this README](https://github.com/tresorsawasawa/MyPortfolio/blob/develop/README.md) to help you how to fill out all the sections. I know you can do that :muscle: :muscle:

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @Stevers1 
Great work on making the changes requested by the previous reviewer  👏👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tresorsawasawa** [comment]
## NOTICE && IMPORTANT
 Kindly make sure you use the same **Pull request** and **branch** for your next review and not **Merge** or **Close** the **Pull request** before it is approved.

**Review by koachcode — APPROVED** [review]
Hi @Stevers1 , 

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Stevers1** [comment]
Good to merge

**Pull request merged**